src/auth/strategy.ts

```diff
@@ -8,56 +8,65 @@ declare global {
     interface User {
       id: number;
       email: string;
-      password_hash: string;
     }
   }
 }
 
-const loadStrategy = () => {
-  passport.use(
-    new LocalStrategy(
-      {
-        usernameField: "email",
-      },
-      async (email, password, cb) => {
-        try {
-          const user = getStudentByEmail(email);
-          if (!user) {
-            return cb(null, false, {
-              message: "Incorrect email or password.",
-            });
-          }
-
-          const match = await bcrypt.compare(password, user.password_hash);
-          if (!match) {
-            return cb(null, false, {
-              message: "Incorrect email or password.",
-            });
-          }
-
-          return cb(null, user);
-        } catch (err) {
-          return cb(err);
+passport.use(
+  new LocalStrategy(
+    {
+      usernameField: "email",
+    },
+    async (
+      email: string,
+      password: string,
+      done: (error: any, user?: Express.User | false) => void
```

Comment:
What could be done here to avoid using `any` as a type? Also why is the user type optional?